example/README.md

```diff
@@ -0,0 +1,14 @@
+# Sample App
+
+To run the sample app, please run:
+
+```
+git clone https://github.com/applegrew/django-select2.git
+cd django-select2/example
+python3 -m pip install -r requirements.txt
```

Comment:
I am not so sure. If you are an experienced user, you will know to wrap this inside a venv. If you are not, you won't have virtualenv installed. So this might make it more difficult.

Reply:
`python3 -m venv py3` works for everyone, no need to have virtualenv installed.
I think users will thank you not noticing too late they installed "more globally" than they wanted.